webpack.config.js

```diff
@@ -41,7 +41,7 @@ module.exports = [
       path.join(__dirname, './images/sparql.png'),
       path.join(__dirname, './favicon.ico'),
       path.join(__dirname, './solid-client-id.jsonld'),
-      path.join(process.cwd(), './queries.json'),
+      path.join(__dirname, './queries.json'),
```

Comment:
I'm not sure this is correct, as this may break overriding queries.json when using `bin/generate.js`.
Can you double-check this? (as we don't have unit tests for this)

Reply:
It seems to still work; using the option `-q` results in custom queries being made available.

![image](https://github.com/user-attachments/assets/22325184-3051-4ec9-bf16-39e766109987)

Reply:
Should I add a unit test for this? And would it be a good start to check if the custom query correctly ends up in `queries.json` in the `build`  folder?

Reply:
Better safe than sorry!

Calling `./bin/generate -q /home/custom_queries` worked correctly. 
The reason is works is that `queries.json` gets generated in the directory denoted by `__dirname` and copied from there to `build/`.

I thought, what is `queries.json` gets generated in `cwd()` which would be incompatible with using `__dirname` in some cases. I testes this in 2 ways.
I ran `bin/generate -q /home/custom_queries` outside of the repo, this failed because it could not resolve `babel-loader`, I assume this is normal behaviour?
I also ran `.bin/generate` from a subdirectory of the repo. This worked correctly.

Do these tests cover everything?

Reply:
> I ran bin/generate -q /home/custom_queries outside of the repo, this failed because it could not resolve babel-loader, I assume this is normal behaviour?

That _should_ also work, but probably doesn't in this case because it can't find the node_modules.
Can you try globally installing this package (your dev version), and running the bin like that? (`yarn link` should also be able to do that (at least yarn classic, not sure about the newer ones))
You should then be able to run `comunica-web-client-generator` from anywhere.

Reply:
This also seemed to work: `comunica-web-client-generator -q /home/custom_queries` works in any directory and I used a print to make sure it's using my local version correctly.

Reply:
I just re-read your previous comment.
The fact that `queries.json` is generated in `__dirname` is a bit strange.
This means that even when running the command elsewhere, `queries.json` will get generated within the directory of the source code.
I would the source code to remain immutable, and only the cwd to get modified.
Do you think it's feasible to restore the old behaviour of this file getting generated in `process.cwd()`? This would also simplify debugging on this file.

Reply:
I will look into it!